content-jumpbox-provisioner/k8s_ssh/k8s_clusters.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo -e \"
+~~~
+
+\e[31mKubernetes Instructions\e[39m
+
+To show available clusters:
+list-clusters
+
+To connect to a cluster:
+source cluster-login.sh upp-delivery-prod-eu
```

Comment:
Probably we also need to instruct them to open the outputted URL in a browser and to copy the token afterwards.